alertmanager-client/shell.nix

```diff
@@ -0,0 +1,35 @@
+{ nixpkgs ? import ../nix/pin.nix {}, compiler ? "default", doBenchmark ? false }:
+
+let
+
+  inherit (nixpkgs) pkgs;
+
+  f = { mkDerivation, alertmanager-openapi, base, http-client, lens
+      , lib, text, transformers, cabal-install
+      }:
+      mkDerivation {
+        pname = "alertmanager-client";
+        version = "0.1.0.0";
+        src = ./.;
+        libraryToolDepends = [ cabal-install ];
+        libraryHaskellDepends = [
+          alertmanager-openapi base http-client lens text transformers
+        ];
+        homepage = "https://github.com/SupercedeTech/alertmanager-client#readme";
+        license = lib.licenses.bsd3;
+      };
```

Comment:
Why not use `callCabal2Nix`?

Reply:
because many, /many/ more packages are broken, I considered going from me not being able to build it at all to having it generate and build the new bindings good enough for this upgrade cycle.